src/main/java/com/spotify/autoscaler/Main.java

```diff
@@ -154,12 +155,15 @@ private Main() throws URISyntaxException, IOException {
     registry.register(APP_PREFIX.tagged("what", "daily-resize-count"),
         (Gauge<Long>) () -> db.getDailyResizeCount());
 
-    registry.register(APP_PREFIX.tagged("what", "failing-cluster-count"),
-        (Gauge<Long>) () -> db.getBigtableClusters()
-                            .stream()
-                            .filter(p -> p.enabled())
-                            .filter(p -> p.consecutiveFailureCount() > 0)
-                            .count());
+      for (ErrorCode code : ErrorCode.values()) {
```

Comment:
Cool, so one Gauge per error code? That should solve the current problem.

Do you think it's possible and easy to add a Gauge to `ClusterStats` with `consecutiveFailureCount` and tagged with the error code + message? Just for more flexibility with what you can do from Alien. Don't have to do it in this PR.